README.md

```diff
@@ -227,6 +227,17 @@ psql -U postgres -d timelessdb_dev
 psql -U timeless_user -d timelessdb_dev
 ```
 
+### Redis Installation
+We use Redis for cache storage. For installing Redis:
+```
+sudo apt install make gcc libc6-dev tcl
+wget http://download.redis.io/redis-stable.tar.gz
+tar xvzf redis-stable.tar.gz
+cd redis-stable
+sudo make install
+src/redis-server > /dev/null &
```

Comment:
@paulodamaso  The `/dev/null &` part was added for rultor not to be stuck in the command.
Users would like to see the output and know everything is okay.
please make it `src/redis-server` - the command makes a good draw in the output anyway 😅